pkg/ranchhand/processhosts.go

```diff
@@ -52,10 +52,9 @@ var (
 			"sudo systemctl start docker",
 		},
 		"rhel": {
-			"sudo subscription-manager repos --enable rhel-7-server-extras-rpms || echo 'Error enabling rhel extras repo, continuing...'",
 			"sudo yum install -y yum-utils device-mapper-persistent-data lvm2",
-			"sudo yum-config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo",
-			"sudo yum install -y docker-ce-18.09.6-3.el7 docker-ce-cli-18.09.6-3.el7 containerd.io",
+			"export REPO_ROOT=\"https://mirrors.domino.tech/artifacts/docker\"",
```

Comment:
Any particular reason to do the replace in shell instead of just in go?

Reply:
If you mean using the shell variable, it seemed like the right confluence of clean and low-effort for this situation.

Reply:
Yeah